src/navigate-pop/navigate-files-folders.md

```diff
@@ -58,6 +58,9 @@ Enter a search term; matched items will automatically populate below the search
 
 Click the down arrow and enter a date in `When` to limit the search to files' creation date. You can also select a relative date range from the list. Choose to apply the date query to `Last Modified`, `Last Used`, or `Created` file attributes.
 
+![Change file path]
```

Comment:
This is the Markdown to add an image, but no image is linked. Are you planning to add one?

Reply:
About the image, I am not able to see them, but I could add one, if the interface being shown in german isnt a problem